acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/pageobjects/BitbucketScmConfig.java

```diff
@@ -23,8 +26,13 @@ public BitbucketScmConfig(Job job, String path) {
     }
 
     public BitbucketScmConfig credentialsId(String credentialsId) {
+        LOGGER.log(Level.INFO, "Zero:");
```

Comment:
I know this is in tests, but maybe we could have a slightly more descriptive log?

Reply:
This was purely for debugging, it will be removed shortly.